### ISSUE TITLE
Rule idea: variable-colon-space-after

### ISSUE THREAD

**Issue by bjankord** [kristerkari/stylelint-scss#51]
This rule idea comes from scss-lint's [Space After Variable Colon rule](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftervariablecolon)

From the scss-lint docs:

_Variables should be formatted with a single space separating the colon from the variable's value._

**Bad: no space after colon**
```scss
$my-color:#fff;
```

**Bad: more than one space after colon**
```scss
$my-color:  #fff;
```

**Good**
```scss
$my-color: #fff;
```

**Comment by dryoma** [comment]
Published with 1.3.0.
